lab_2_keywords_cooccurrence/main.py

```diff
@@ -141,7 +252,16 @@ def calculate_cumulative_score_for_candidates_with_stop_words(candidate_keyword_
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (check_list(candidate_keyword_phrases, tuple, False) and check_dict(word_scores, str, float, False)
+            and check_list(stop_words, str, False)):
+        return None
+    candidates_cumulative_score = {}
+    for phrase in candidate_keyword_phrases:
+        candidates_cumulative_score[phrase] = 0
+        for word in phrase:
+            if word not in stop_words:
+                candidates_cumulative_score[phrase] += int(word_scores[word])
```

Comment:
why int?

Reply:
without it i got an error in github (in check types, i think)

Reply:
by converting it to int you lose precision, as scores are initially float 

can you give me more details on the error you get otherwise?